python-lib/dku_google/clusters.py

```diff
@@ -232,6 +232,7 @@ def with_initial_node_count(self, node_count):
         return self
     
     def with_labels(self, labels={}):
+        labels = {label["from"]: label.get("to", "") for label in labels}
```

Comment:
do we want to raise an error explicitly if the label key (from) is empty?

Reply:
So apparently, the form will not let you save an entry with no value. If you try to cheat and put multiple entries, one without key and one with, the entry without value will disappear and not be saved.
So I don't think we can really end up in that situation, but some defensive coding can't hurt